CHANGELOG.md

```diff
@@ -28,6 +28,10 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
 
 ### Formatter
 
+### Bug fixes
+
+- Fix [#4121](https://github.com/biomejs/biome/issues/4326), don't ident css selector when comments before. Contributed by @fireairforce
```

Comment:
```suggestion
- Fix [#4121](https://github.com/biomejs/biome/issues/4326), don't ident CSS selector when comments before. Contributed by @fireairforce
```
You need to add something after the word "before" because, as is, it's incomplete

Reply:
done~ updated the changelog